src/rest/zone_client.erl

```diff
@@ -0,0 +1,175 @@
+%%%-------------------------------------------------------------------
+%% @doc
+%%  Enables zones to communicate with each other
+%%
+%%  Declared functions are assigned one out of 5 categories:
+%%   1. necessary -- necessary to make naive engine
+%%   2. important -- maybe necessary, I'm not sure yet
+%%   3. optional  -- maybe useful, but I wouldn't focus on them
+%%   4. ignore    -- they will be needed for 'indexed' version
+%%   5. ???       -- I'm not sure if they need to exist at all
+%% @end
+%%%-------------------------------------------------------------------
+-module(zone_client).
+-author("pawel").
+
+%% API
+-export([
+    healthcheck/1, %% @todo ???
```

Comment:
Healtcheck jest potrzebny do kubernetesa, służy do sprawdzenia czy zona dobrze działa, chociaż to bardziej po stronie serwera. W każdym razie - chyba to jest opcjonalne jak narazie

```suggestion
    healthcheck/1, %% @todo optional
```